mozilla-mobile/android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -126,7 +126,10 @@
     <!-- Title for using the suggest strong password confirmation dialog -->
     <string name="mozac_feature_prompts_suggest_strong_password_title">Use strong password?</string>
     <!-- Content description for the suggest strong password confirmation dialog -->
-    <string name="mozac_feature_prompts_suggest_strong_password_description">Protect your accounts by using a strong, randomly generated password. Get quick access to your passwords by saving it in your account.</string>
+    <string name="mozac_feature_prompts_suggest_strong_password_description" moz:removedIn="130" tools:ignore="UnusedResources">Protect your accounts by using a strong, randomly generated password. Get quick access to your passwords by saving it in your account.</string>
+    <!-- Content description for the suggest strong password confirmation dialog -->
+    <string name="mozac_feature_prompts_suggest_strong_password_description_2">Protect your accounts by using a strong, randomly generated password. It\'ll be saved into your account for future use.</string>
```

Comment:
@Alexandra-Virvara I see you introduced this string. Can you please update to the [typographic ("curly") apostrophe](https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-apostrophes-and-quotation-marks-65)?